In-house patch to remove unnecessary dependencies from Swift's
requirements files. The specific reasons are as follows:

greenlet	Not applicable

--- swift-2.2.0/requirements.txt
+++ swift-2.2.0/requirements.txt
@@ -4,7 +4,6 @@
 
 dnspython>=1.9.4
 eventlet>=0.9.15
-greenlet>=0.3.1
 netifaces>=0.5,!=0.10.0,!=0.10.1
 pastedeploy>=1.3.3
 simplejson>=2.0.9
--- swift-2.2.0/swift.egg-info/requires.txt
+++ swift-2.2.0/swift.egg-info/requires.txt
@@ -1,6 +1,5 @@
 dnspython>=1.9.4
 eventlet>=0.9.15
-greenlet>=0.3.1
 netifaces>=0.5,!=0.10.0,!=0.10.1
 pastedeploy>=1.3.3
 simplejson>=2.0.9
